doc/solverinterface.rst

```diff
@@ -107,6 +107,10 @@ All abstract model types inherit from the abstract type ``AbstractMathProgModel`
     with licensing restrictions on the number of concurrent runs.
     Users must discard the model object after this method is invoked.
 
+.. function:: copy(m::AbstractMathProgModel)
```

Comment:
Yep, that is new, all the solvers had the copy function for their on Model types, but there was nothing for `AbstractMathProgModel` in MathProgBase. There was even some discussion about what to do with callbacks, I decided to no copying them and warning the user. See JuliaOpt/Gurobi.jl#78